### PR TITLE
feat(component): add margin and write the rest of the missing attributes

### LINKED ISSUE #2: Button Component
https://vercel.com/geist/button

### PR DESCRIPTION
**Pull request by daridjcm**

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ export default function Button({ children, type, disabled }: ButtonProps) {
     <>
       <button
         disabled={disabled}
-        className="bg-blue-500 rounded-xl text-white py-2 px-4 disabled:bg-gray-600"
+        className="bg-blue-500 rounded-xl text-white m-1 py-2 px-4 disabled:bg-gray-600"
```
Comment (Michael-Liendo) [inline src/components/Button.tsx]: Please remove the margin, we have to add spacing outside the component, maybe with classnames or spacing component

**Review by Michael-Liendo — CHANGES_REQUESTED** [review]

**Comment by Michael-Liendo** [comment]
Please run `npm run format`

**Review by daridjcm — COMMENTED** [review]
Done.

File: src/components/Button.tsx (hunk 1 above)
Comment (daridjcm) [reply src/components/Button.tsx]: Okay, you're right.

**Review by Michael-Liendo — APPROVED** [review]
LGTM

**Pull request merged**